### PR TITLE
Env Support for APM Dashboards

### PR DESCRIPTION
**Pull request by kimsauce**
Tracing/Environment Support for APM Dashboards

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/application-components.md [hunk 1]
```diff
@@ -271,13 +267,53 @@ Explore is an out-of-the-box Sumo Logic navigation tool that provides an intuiti
 The **Application Components - Database System Overview** dashboard appears in the window on the right. The dashboard provides an at-a-glance view of the different database engines present in the database component and insights on errors occurring across different engines.
 1. And then keep on going down the hierarchy to specific entities for each of the components to view dashboards at more granular levels.
 
+### Dashboard Examples
+
```
Comment (kimsauce) [inline docs/observability/application-components.md:271]: @JKashyap96 , Is the Application Components Solution related to the APM Dashboards? I've inserted your screenshots here for the time being.

**Review by kimsauce — COMMENTED** [review]
@JKashyap96 , can you take a look at this rough draft? The Application Components Solution doc seems to be related. Can you confirm?

File: blog-service/2022-11-09-traces.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - apm
+hide_table_of_contents: false
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Update - Tracing dashboards now offer native support for monitoring OpenTelemetry using the `deployment.environment` standard tag.
```
Comment (pbrzoska-sumo) [inline blog-service/2022-11-09-traces.md:17]: native support for OpenTelemetry's

File: blog-service/2022-11-09-traces.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+OpenTelemetry `deployment.environment` is a helpful way of slicing APM data between environments (e.g. dev vs prod). Previously this was supported in tracing metrics and available for custom queries and dashboards, and now it's supported in our out-of-the-box dashboards.
```
Comment (pbrzoska-sumo) [inline blog-service/2022-11-09-traces.md:19]: >Previously this was supported

It is already supported as metadata

Comment (pbrzoska-sumo) [reply blog-service/2022-11-09-traces.md:19]: >in our out-of-the-box dashboards.

in our out-of-the-box dashboards as well.

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 4]
```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
...
 ---
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
 ## Application Service Dashboards
 
 Service and Application dashboards are available as two Explore hierarchies. You can access them in the **Explore By** drop down menu.
 
-The **Application Service View** groups services into higher-level applications based on the `application=[app-name]` custom tag that you may add to your tracing data if you want to leverage it fully. If not present, all services will belong to a "default" application. See [how to modify the attributes of your spans](../advanced-configuration/working-with-span-attributes.md). The third level shows the top 50 most active operations executed on the service.
+The **APM: Application View** groups services into higher-level applications based on the `application=[app-name]` custom tag that you may add to your tracing data if you want to leverage it fully. Without the tag, all services will belong to a "default" application. The third level shows the top 50 most active operations executed on the service. See [Working with Span attributes](../advanced-configuration/working-with-span-attributes.md) to learn how to how to customize your span attributes.
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:14]: The **APM: Application View** groups services into higher-level applications based on the `application=[app-name]` custom tag that you may add to your tracing data if you want to leverage it fully. Without the tag, all services will belong to a "default" application. The third level shows environment automatically picked from OT deployment.environment tag. Without the tag, all data will belong to a "default" environment. 
The fourth and last shows the top 50 most active operations executed on the service. See [Working with Span attributes](../advanced-configuration/working-with-span-attributes.md) to learn how to how to customize your span attributes.

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 5]
```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
...
 
-The **Service Application View** takes a contrary approach, displaying services by top level and breaking down their health by application. This view is especially useful for shared services that support more than one application. The third level shows the top 50 most active operations performed on the selected service and application.
+The **APM: Service View** takes a contrary approach, displaying services by top level and breaking down their health by application. This view can be useful for shared services that support more than one application. The third level shows the top 50 most active operations performed on the selected service and application.
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:16]: >The third level shows the top 50 most active operations performed on the selected service and application.

The fourth...

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 6]
```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
...
 
-![Service Dashboards from traces.png](/img/traces/Service-Dashboards-from-traces.png)
+The **APM: Environment View** displays environments at the top level and breaks down application health by environment (i.e., prod or dev). This view can be especially useful for understanding the top-down hierarchy of applications and services in a particular environment. The third level shows the top 50 most active operations executed on the environment, application, and service. <br/><img src={useBaseUrl('img/traces/Service-Dashboards-from-traces.png')} alt="Service Dashboards from traces" width="350"/>
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:18]: >The third level shows the top 50 most active operations performed on the selected service and application.

The fourth...

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 7]
```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
...
 
-* **Latency**: (real-time average) the average time it takes for entry spans for that service to complete. Apart from average you can also select p99, p95, p90, p50 options in service_latency_type filter to view other percentile aggregations.
-
-    ![service-latency.png](/img/traces/service-latency.png)
+* **Latency**: (real-time average) the average time it takes for entry spans for that service to complete. Apart from average you can also select p99, p95, p90, p50 options in service_latency_type filter to view other percentile aggregations.<br/> ![service-latency.png](/img/traces/service-latency.png)
 * **Requests**: (real-time counter) the number of entry spans reported by the service
 * **Errors**: (real-time counter) the number of entry spans for the service that finished with an error
 
-By clicking on any data point on the chart, just like for any other Sumo Logic dashboard, you can view the side panel's **Infrastructure** tab to drill down to related metrics or traces for the selected service.
+By clicking on any data point on the chart, you can view the side panel's **Infrastructure** tab and drill down to related metrics or traces for the selected service.
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:30]: >the side panel's **Infrastructure** tab

the side panel's **Entites** tab

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 8]
```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
...
 
 :::note
 Drilling to traces by application and to logs by both service and application are coming soon and are not working yet.
 :::
 
 ![Explore coffee app to drill down on infrastructure tab.png](/img/traces/coffee-app-infrastructure-tab.png)
 
-## Install the App
+## Installing the Tracing App
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:38]: >## Installing the Tracing App

## Installing the Tracing App (Optional)

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 9]
```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
...
-    ![filter-servicemap.png](/img/traces/filter-servicemap.png)
+The **Service Map** panel displays a service map for filtered applications and services, with the ability to explore the environment.
 
+To add a **Service Map** panel to your Dashboard:
+1. Open or create a Dashboard.
+1. Click **Add Panel** and select **Service Map**. Or select the **Service Map** option on a new Dashboard.<br/>  ![add-servicemap.png](/img/traces/add-servicemap.png)
+    A panel configuration page opens.<br/>  ![create-servicemap.png](/img/traces/create-servicemap.png)
+1. Select from the drop-down menus to filter the Service Map by the following:<br/>  ![filter-servicemap.png](/img/traces/filter-servicemap.png)
    * **Application** if your tracing data has the `application=[app-name]` tag
    * **Service**
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:72]: deployment.environment has been added here

File: docs/observability/application-components.md [hunk 10]
```diff
@@ -271,13 +267,53 @@ Explore is an out-of-the-box Sumo Logic navigation tool that provides an intuiti
 The **Application Components - Database System Overview** dashboard appears in the window on the right. The dashboard provides an at-a-glance view of the different database engines present in the database component and insights on errors occurring across different engines.
 1. And then keep on going down the hierarchy to specific entities for each of the components to view dashboards at more granular levels.
 
+### Dashboard Examples
```
Comment (pbrzoska-sumo) [inline docs/observability/application-components.md:270]: Wrong view. This is APM:* not Application components

**Review by pbrzoska-sumo — CHANGES_REQUESTED** [review]

**Comment by JKashyap96** [comment]
Changes suggested by Pawel needs to be resolved.

File: blog-service/2022-11-09-traces.md (hunk 2 above)
Comment (kimsauce) [reply blog-service/2022-11-09-traces.md:17]: Is this right @pbrzoska-sumo ?

Update - Tracing dashboards now offer native support for OpenTelemetry's `deployment.environment` standard tag.

File: blog-service/2022-11-09-traces.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Update - Tracing dashboards now offer native support for OpenTelemetry's `deployment.environment` standard tag.
+
+OpenTelemetry `deployment.environment` is a helpful way of slicing APM data between environments (e.g. dev vs prod). Previously, this was supported in tracing metrics as metadata and available for custom queries and dashboards, and now it's supported in our out-of-the-box dashboards as well.
```
Comment (kimsauce) [inline blog-service/2022-11-09-traces.md:19]:
```suggestion
OpenTelemetry `deployment.environment` is a helpful way of slicing APM data between environments (e.g. dev vs prod). Previously, this was supported in tracing metrics as metadata and available for custom queries and dashboards, and now it's supported in our out-of-the-box dashboards.
```

**Review by pbrzoska-sumo — COMMENTED** [review]
not sure what's the purpose of [static/img/release-notes/service/tracing-panels.gif] ?

**Comment by kimsauce** [comment]
> not sure what's the purpose of [static/img/release-notes/service/tracing-panels.gif] ?

Good catch -- was a placeholder I'd forgotten to remove. Fixed this.

**Comment by JKashyap96** [comment]
@kimsauce Where can I see how the updated doc will look before you merge? A few other screenshots other than the dashboards might need updates as well.

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 12]
```diff
@@ -84,26 +81,73 @@ You are limited to three Trace List panels in a dashboard.
...
+  | Duration Breakdown | ![breakdown](/img/traces/breakdown.png) | Each color indicates a service. The colors assigned to services are always the same on your account. You can change the color in the span summary tab after clicking on the individual span in trace view.<br/>Hover over to view a percentage breakdown of how long each span covers in the trace.<br/>![img](/img/traces/span-hover-view.png) |
+  | Number of errors | 0 | The number of errors in the trace. |
+  | Status | 200 | The HTTP status code of the trace. A menu is available in this column when hovering on a row. The menu has an option to **Show similar traces**.<br/>![img](/img/traces/similar-traces-menu.png) |
+
+8. Click the **General** tab to edit the Panel Details. Enter a name for the panel, set a **Title Font Size**, and add a short **Description**.<br/>  ![tracelist-details.png](/img/traces/tracelist-details.png)
+9. Click **Add to Dashboard**. The panel loads in your Dashboard to review all traces according to the query.<br/>  ![panel-trace.png](/img/traces/panel-trace.png)
+
+
+### Dashboard Examples
```
Comment (kimsauce) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:108]: @JKashyap96 Moved these dashboards here from App Components. Does this look ok?

Comment (JKashyap96) [reply docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:108]: Yes

File: docs/apm/traces/working-with-tracing-data/tracing-dashboards.md [hunk 13]
```diff
@@ -1,44 +1,41 @@
...
+The **Tracing - Application Services Health** app is automatically installed for all users of your organization once Sumo Logic detects OpenTelemetry-compatible tracing data coming from your instrumented services. The content is placed in **Sumo Logic Tracing - default dashboards** inside the **Admin Recommended** folder and is automatically available for all users in the organization.
 
 :::note
-Do not modify or delete content in the **Admin Recommended** folder as it is maintained and updated automatically. If for any reason this gets removed, you can install the App manually from App Catalog. 
+Do not modify or delete content in the **Admin Recommended** folder as it is maintained and updated automatically. If for any reason this gets removed, you can install the App manually from App Catalog.
 :::
 
 To install the app, do the following:
```
Comment (JKashyap96) [inline docs/apm/traces/working-with-tracing-data/tracing-dashboards.md:41]: SS in line 43 needs to updated as well. @kimsauce SS is in : https://sumologic.slack.com/archives/C04BC5J7Q20/p1668635033883979

**Comment by kimsauce** [comment]
@JKashyap96 fixed screenshots - can you take one last look?

**Review by JKashyap96 — APPROVED** [review]
LGTM

**Pull request merged**